posthog/test/test_filter_model.py

```diff
@@ -137,6 +137,21 @@ def test_user_properties(self):
         self.assertEqual(events[0], event2)
         self.assertEqual(len(events), 1)
 
+    def test_cohort_properties(self):
```

Comment:
`test_person_cohort_properties` or something more specific to denote that this is cohort filtering for person queries and not, for example, event cohort filtering